ui/admin/app/styles/app.scss

```diff
@@ -886,6 +886,10 @@
 // Session Recording sidebar details
 .rose-layout-body-content-sidebar .section-details {
   margin-bottom: 3rem;
+
+  .rose-metadata-list-item {
```

Comment:
Is this something that should be in the component css? How come we're adding it here?

Reply:
Moved it to the component. The fonts were too big, and the dates were getting cut off